prompt_prototype.rb

```diff
@@ -41,11 +41,23 @@ def morph_records(scraper, query)
   JSON.parse(RestClient.get(url), symbolize_names: true).map { |d| [d[:id], d]  }.to_h
 end
 
-wikidata_records = wikidata_sparql("SELECT ?item ?itemLabel WHERE { ?item wdt:P39 wd:#{wikidata_membership_item}.  SERVICE wikibase:label { bd:serviceParam wikibase:language \"[AUTO_LANGUAGE],en\". } }")
-morph_records = morph_records(morph_scraper, scraper_sql)
-
 country_slug, house_slug = ep_country_and_house.split('/')
 popolo = Everypolitician::Index.new.country(country_slug).legislature(house_slug).popolo
+
+house_wikidata_id = popolo.organizations.find { |o| o.classification == 'legislature' }.wikidata
+sparql = %(
+SELECT ?membership ?membershipLabel ?item ?itemLabel WHERE {
+  # Find any memberships that are "part of" (P361) the house
+  ?membership wdt:P361 wd:#{house_wikidata_id} .
+  # Find any items that have a "position held" (P39) of any house membership
+  ?item wdt:P39 ?membership .
+  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
```

Comment:
Just a question here - this is a looser query than the SPARQL @tmtmtmtm suggested in Slack, which also had:

```
?membership wdt:P279* wd:Q4175034
```

... as a constraint. Is there a particular reason not to add that as well?